resources/istio-configuration/README.md

```diff
@@ -0,0 +1,42 @@
+# Istio Configuration
+
+## Overview
+
+[Istio](https://istio.io/) is an open-source service mesh providing a uniform way to integrate microservices, manage traffic flow across microservices, enforce policies, and aggregate telemetry data.
+
+The documentation here is for developers only, please follow the installation instructions from [istio.io](https://istio.io/docs/setup/install/istioctl/) for all other use cases.
+
+The Istio Configuration Helm chart consists of:
+
+- `istio-operator.yaml` file implementing Kyma-specific changes and configuration options,
+- Mutual TLS (mTLS) configuration enabling mTLS cluster-wide in a STRICT mode,
+- Istio monitoring configuration details.
+
+This chart can install the following Istio components:
+
+- ingressgateway
+- egressgateway
+- istiod (pilot, citadel, and galley)
+
+To enable or disable each component, change the corresponding `enabled` flag.
+
+## Installation
+
+Installation of the Istio Operator requires [Reconciler](https://github.com/kyma-incubator/reconciler/tree/main/pkg/reconciler/instances/istio). Reconciler uses `istioctl` and a rendered `istio-operator.yaml` file to install Istio on a cluster. To install the component run:
+
+```bash
+kyma deploy --components istio-configuration@istio-system
+```
+
+## Configuration
+
+The installation of Istio Operator ships with reasonable defaults. There may be circumstances in which defaults require overrides.
```

Comment:
What does "reasonable" mean? That the defaults match most use cases? As a reader, I would hope that any defaults are not unreasonable :D 

For "overrides", does it mean that users will change the default values in the values.yaml? I'm asking because we tried to get away from "overrides" as a noun.

```suggestion
The installation of Istio Operator ships with reasonable defaults. There may be circumstances in which you want to change the defaults.
```

Reply:
I will remove "reasonable" and rephrase.

Reply:
Yes, they can change the default configuration as described in the **Change Kyma settings** document.